platform/default/include/mbgl/gfx/headless_frontend.hpp

```diff
@@ -17,6 +18,11 @@ class TransformState;
 
 class HeadlessFrontend : public RendererFrontend {
 public:
+    struct RenderResult {
```

Comment:
shall we add `bool isValid() const` here?

Reply:
Yeah could be added. In current use std::exception is thrown if an error is encountered